etc/bash_completion.d/yab

```diff
@@ -0,0 +1,16 @@
+# bash completion for yab executable
+have yab &&
```

Comment:
nit: newline (probably a couple) separating the header from the body

Reply:
Done